src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -643,6 +844,7 @@ public void updateSubscribers()
         isBuildingsDirty = false;
         permissions.clearDirty();
 
+        // TODO: look into potentially keeping lists as well
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)